src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private final double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(getArea() + radius + "color:" + color);
+    }
+
+    @Override
+    public Color getColor() {
+        return color;
+    }
```

Comment:
this can be extracted to a parent class, it's common for all subclasses since color is a field in superclass